client/flutter/ios/Runner/Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>NSLocationAlwaysAndWhenInUsageDescription</key>
 	<string>Your location is used to deliver you the most relevant information and push notifications to your area.</string>
 	<key>CFBundleDisplayName</key>
-	<string>COVID-19</string>
+	<string>WHO COVID-19</string>
```

Comment:
Is this being changed on Android too? And is that still short enough to not be cut off by the IOS launcher?

Reply:
This is just a merge so we shouldn;'t change anythig here